### PR TITLE
167408611 venue design system

### PR DESCRIPTION
**Pull request by SeriousHorncat**
Venue page with the initial draft of the design system in place

### CODE REVIEW

File: src/views/Travel/VenueLocation.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <Tile bgcolor="bg-white" title="Venue" titleColor="text-wri-dark-blue"
+    >>
```
Comment (RBangel) [inline src/views/Travel/VenueLocation.vue:3]: double closing angle bracket. I think you meant for there to only be one.

File: src/views/Travel/Food.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<template>
+  <Tile bgcolor="bg-white" title="Cuisine and Nightlife" titleColor="text-wri-dark-blue">
+    <div class="md:flex md:justify-center">
+      <div class="md:flex-shrink-0 md:w-1/2 md:max-w-xl">
+        <img src="../../assets/nightlife.jpg" />
```
Comment (RBangel) [inline src/views/Travel/Food.vue:5]: Same picture as recreation

File: src/views/Travel/Recreation.vue [hunk 3]
```diff
@@ -0,0 +1,79 @@
+<template>
+  <Tile bgcolor="bg-white" title="Recreation" titleColor="text-wri-dark-blue">
+    <div class="p-6 md:flex md:justify-center">
+      <div class="md:flex-shrink-0 md:w-1/2 md:max-w-xl">
+        <img src="@/assets/nightlife.jpg" />
```
Comment (RBangel) [inline src/views/Travel/Recreation.vue:5]: same picture as food

**Review by RBangel — APPROVED** [review]
I feel like you're making your life more difficult than it needs to be by relying on `h1`, `h2`, `h3`, and `p` tags instead of just having class based `div` tags, but maybe I'm mistaken.

**Comment by SeriousHorncat** [comment]
I defaulted to using the html tags since seemed alright at the time.  I wouldn't mind going back through and making it class-based with divs instead though.  :3

**Comment by SeriousHorncat** [comment]
Actually, I am thinking we could just leave it as is for now and after we get it released to production, we refactor it and standardize the CSS text and include it across the site?  @RBangel

**Pull request merged**